aws/credentials/stscreds/assume_role_provider.go

```diff
@@ -9,7 +9,7 @@ to refresh the credentials will be synchronized. But, the SDK is unable to
 ensure synchronous usage of the AssumeRoleProvider if the value is shared
 between multiple Credentials, Sessions or service clients.
 
-Assume Role
+# Assume Role
```

Comment:
This seem to be getting added by gofmt? I am not sure why these wouldn't already be present, but I did confirm that `gofmt` wants to add these now